cterasdk/edge/config.py

```diff
@@ -35,6 +36,7 @@ def set_hostname(self, hostname):
         """
         Set the hostname of the gateway
 
+        :param str hostname: New hostname to set
         :return str: The command response
```

Comment:
The set_hostname command returns the hostname that was set (as a str)

Reply:
Fixed